src/database.js

```diff
@@ -4,46 +4,76 @@ const {MongoClient} = require("mongodb")
         useNewUrlParser: true,
         useUnifiedTopology: true
     });
 
-    if (!database) {
-        database = connection.db();
+    database = connection.db();
 
-        await database.collection("users").insertMany([
-                {
-                    id: 1,
-                    firstName: "Kirill",
-                    password: "123"
-                    // secondName: "Lazarev",
-                    // email: "lazeratrax@gmail.com",
-                    // password: "1234567Q",
-                    // tel: 375447610553,
-                    // avatarUrl: "https://econet.ru/uploads/pictures/456173/content_199820.jpg",
-                    // isOnline: false
-                },
-                {
-                    id: 12,
-                    firstName: "Lena",
-                    password: "123"
-                    // secondName: "Lazareva",
-                    // email: "kate@gmail.com",
-                    // password: "1234567Q",
-                    // tel: 375298887777,
-                    // avatarUrl: "https://cs11.livemaster.ru/storage/topicavatar/600x450/04/66/4078a444edc286bd35b0a0a67c680cc42204sw.jpg?h=AH8vAuYPoFKwWsdzxggafA",
-                    // isOnline: false
-                }
-            ]
-        )
-    }
-    return database
+    await database.collection("users").insertMany([
+        {
+            id: 1,
+            name: "Kirill",
+            password: "12345",
+            email: "lazeratrax@gmail.com",
+            avatar: "https://econet.ru/uploads/pictures/456173/content_199820.jpg",
+            timeZone: 10,
+            isPasswordSet: false,
+            isTeacherPreview: false,
+            access_token: String,
+            // isOnline: false
+        },
+        {
+            id: 12,
+            name: "Lenaa",
+            password: "12345",
+            // email: "lazeratrax@gmail.com",
+            // avatar: "https://econet.ru/uploads/pictures/456173/content_199820.jpg",
+            // timeZone: 10,
+            // isPasswordSet: false,
+            // isTeacherPreview: false,
+            access_token: String,
+            // isOnline: false
+        }
+    ]);
+    await database.collection("articles").insertMany([
+        {
+            id: 1001,
+            title: 'firstName',
+            content: ' value',
+            illustration: "https://econet.ru/uploads/pictures/456173/content_199820.jpg",
+            author: 1
+        },
+        {
+            id: 1002,
+            title: 'Name',
+            content: 'Invalid',
+            illustration: "https://econet.ru/uploads/pictures/456173/content_199820.jpg",
+            author: 12
+        },
+    ]);
+    //для валидаций
+    // await database.collection("Output").insertMany([
+    //     {
+    //         param: 'firstName',
+    //         msg: 'Invalid value',
+    //     },
+    //     {
+    //         param: 'password',
+    //         msg: 'Invalid value',
+    //     }
+    // ])
```

Comment:
Better to create other function for this (like `insertInitialMockData`)

Reply:
ok, I must use any service, like https://www.mockaroo.com/ ? And insert generated data into 
"await database.collection("users").insertMany(["   ])"" ?

Reply:
It isn't required. Only move this logic to separate function. Code style issue